index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>판다마켓</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <header>
+      <a href="index.html">
```

Comment:
💊 제안
요구사항인 `"판다마켓" 클릭 시 루트 페이지('/')로 이동합니다.`에 따라 아래처럼 작성하시는 것을 추천드립니다.

```suggestion
      <a href="/">
```